x-pack/plugins/siem/public/management/pages/policy/view/policy_list.tsx

```diff
@@ -89,63 +95,116 @@ export const PolicyList = React.memo(() => {
-          defaultMessage: 'Version',
+        field: 'created_at',
+        name: i18n.translate('xpack.siem.endpoint.policyList.createdAt', {
+          defaultMessage: 'Created Date',
         }),
-        render(pkg) {
-          return `${pkg.title}  v${pkg.version}`;
+        render(createdAt: string) {
+          return <FormattedDateAndTime date={new Date(createdAt)} />;
         },
       },
       {
-        field: 'description',
-        name: i18n.translate('xpack.siem.endpoint.policyList.descriptionField', {
-          defaultMessage: 'Description',
+        field: 'updated_by',
+        name: i18n.translate('xpack.siem.endpoint.policyList.updatedBy', {
+          defaultMessage: 'Last Updated By',
         }),
         truncateText: true,
       },
       {
-        field: 'config_id',
-        name: i18n.translate('xpack.siem.endpoint.policyList.agentConfigField', {
-          defaultMessage: 'Agent Configuration',
+        field: 'updated_at',
+        name: i18n.translate('xpack.siem.endpoint.policyList.updatedAt', {
+          defaultMessage: 'Last Updated',
         }),
-        render(version: string) {
-          return (
-            <LinkToApp
-              data-test-subj="agentConfigLink"
-              appId="ingestManager"
-              appPath={`#/configs/${version}`}
-              href={`${services.application.getUrlForApp('ingestManager')}#/configs/${version}`}
-            >
-              {version}
-            </LinkToApp>
-          );
+        render(updatedAt: string) {
+          return <FormattedDateAndTime date={new Date(updatedAt)} />;
         },
       },
+      {
+        field: 'package',
+        name: i18n.translate('xpack.siem.endpoint.policyList.versionFieldLabel', {
+          defaultMessage: 'Version',
+        }),
+        render(pkg) {
+          return i18n.translate('xpack.siem.endpoint.policyList.versionField', {
+            defaultMessage: '{title} v{version}',
+            values: {
+              title: pkg.title,
+              version: pkg.version,
+            },
+          });
+        },
+      },
+      {
+        field: '',
+        name: 'Actions',
+        actions: [
+          {
+            // eslint-disable-next-line react/display-name
+            render: (item: Immutable<PolicyData>) => {
+              return (
+                <EuiContextMenuItem icon="link" key="datasourceEdit">
+                  <LinkToApp
+                    data-test-subj="agentConfigLink"
+                    appId="ingestManager"
+                    appPath={`#/configs/${item.config_id}`}
+                    href={`${services.application.getUrlForApp('ingestManager')}#/configs/${
+                      item.config_id
+                    }`}
+                  >
+                    <FormattedMessage
+                      id="xpack.siem.endpoint.policyList.agentConfigAction"
+                      defaultMessage="View Agent Config"
```

Comment:
Forgot this. consider:

```suggestion
                      defaultMessage="View Agent Configuration"
```

Since that is how Ingest refers to it (currently 😬 )